python/common/org/python/types/Bool.java

```diff
@@ -302,7 +311,7 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
                 return new org.python.types.Int(0);
             } else {
                 if (org.Python.VERSION < 0x03060000) {
-                    return new org.python.types.Bool(false);
+                    return (org.python.types.Bool) FALSE;
```

Comment:
The cast shouldn't be needed here (and the constant should be fully qualified to avoid ambiguity)

Reply:
Done!